src/main/java/datawave/query/util/MetadataHelper.java

```diff
@@ -988,9 +993,18 @@ public Type<?> getDatatypeFromClass(Class<? extends Type<?>> datatypeClass) thro
      */
     @Cacheable(value = "getTermFrequencyFields", key = "{#root.target.auths,#root.target.metadataTableName,#p0}", cacheManager = "metadataHelperCacheManager")
     public Set<String> getTermFrequencyFields(Set<String> ingestTypeFilter) throws TableNotFoundException {
+        
         Multimap<String,String> termFrequencyFields = loadTermFrequencyFields();
         
-        return getFields(termFrequencyFields, ingestTypeFilter);
+        Set<String> fields = new HashSet<>();
```

Comment:
it looks like the getFields helper method was lost.

Reply:
yeah.  I had to revert these changes made by @mineralntl because it breaks the datawave build.  He has a datawave branch which fixes the test failures, but I requested some additional changes with my review.  I'm not sure what the status of that is, but if his PR gets merged soon, I will revert the revert.  Thoughts @mineralntl ?

Reply:
@mineralntl if you have addressed all the feedback in that PR, I can review it again.